app/page.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+const PageMoved = () => {
+  return (
+      <div>
+        <h1>Page Moved</h1>
```

Comment:
Maybe instead of this, we could have a selector so you can pick the app you want to use.  Then there's less typing in the url bar.